ykllvmwrap/src/symbolizer.rs

```diff
@@ -8,21 +8,21 @@ use std::{
 };
 
 extern "C" {
-    fn ykllvm_symbolzer_new() -> *mut c_void;
-    fn ykllvm_symbolzer_free(symbolizer: *mut c_void);
-    fn ykllvm_symbolzer_find_code_sym(symbolizer: *mut c_void, addr: usize) -> *mut c_char;
+    fn ykllvmwrap_symbolzer_new() -> *mut c_void;
```

Comment:
I assume this is meant to be "symbolizer" with a full fat "i"?

Also, I'm not sure whether we need to use the whole prefix here: I'd be OK with just `yk_` as a prefix (we can trust ourselves to keep these unique across the yk packages) if you'd prefer it.

Reply:
I prefer to "namespace" them so that they can't be confused with our API.

I'd be OK with `__yk_` though. That makes is pretty clear it's internal.

Reply:
Fine with me.